src/api/baseAPI.ts

```diff
@@ -18,10 +18,11 @@ export async function baseAPI<T>({
     },
     body: body ? JSON.stringify(body) : null,
   });
+
   if (!result.ok) {
-    throw new Error('서버에서 에러가 발생했습니다.');
+    throw new Error('요청에 실패하였습니다.');
   }
 
-  if (method === 'GET') return result.json();
-  return null;
+  if (result.status === 204) return null;
+  return result.json();
```

Comment:
early return으로 보호구문을 잘 짰네! 😊

그런데 if문의 컨벤션을 통일시키는 건 어떨까? 
{} 를 써서 return 문을 감싸도 좋을 것 같고, 아니면 throw new Error를 바로 하는 건 어때?? 🤔
```suggestion
  if (result.status === 204){ 
    return null;
  }
  
  return result.json();
```